test/valgrind/suppressions/geos_c_impl_ruby.supp

```diff
@@ -0,0 +1,16 @@
+{
+  <ruby_gc_false_positive>
+  Memcheck:Addr8
+  ...
+  fun:gc_marks
+  fun:gc_start
+  ...
+  fun:main
+}
+{
+  <ruby_flo_to_s_false_positive>
+  Memcheck:Overlap
+  fun:__memcpy_chk
+  fun:memmove
+  fun:flo_to_s
+}
```

Comment:
Do we need this in version control or is this the output of a test?

Reply:
We need this, it is a _suppression file_. Ruby has some memory issues itself because it doesn't clean up everything at exit. This file tells memcheck that those issues are to be ignored.

The second one by the way seems like a real ruby bug, but I didn't find a small reproducible case to open an issue in the ruby bug tracker..

Reply:
That makes sense thanks